### PR TITLE
add BadgeTitle

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/components/badges/BadgeTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <ExternalLink url={getEtherscanAddressUrl(derivativeAddress)}>
+        <ContractName address={derivativeAddress} />
+      </ExternalLink>
+    )
+
+  if (!proof) return null
+
+  return (
+    <>
+      <ProofName badge={proof} /> (derivative)
```
Comment (T-Damer) [inline src/components/badges/BadgeTitle.tsx:33]: We also have `(email)` text in designs
image

Comment (T-Damer) [reply src/components/badges/BadgeTitle.tsx:33]: Also, what will this function return if you pass `derivativeAddress`, but won't pass `proof`?

**Review by T-Damer — APPROVED** [review]
only small fixes needed

File: src/components/badges/BadgeTitle.tsx (hunk 1 above)
Comment (MixailE) [reply src/components/badges/BadgeTitle.tsx:33]: For simplicity added just return of the name of the derivative contract

**Pull request merged**